pkg/master/master.go

```diff
@@ -570,16 +570,30 @@ func (m *Master) init(c *Config) {
 	requestInfoResolver := &apiserver.APIRequestInfoResolver{APIPrefixes: sets.NewString(strings.TrimPrefix(defaultVersion.Root, "/")), RestMapper: defaultVersion.Mapper}
 	apiserver.InstallServiceErrorHandler(m.handlerContainer, requestInfoResolver, apiVersions)
 
+	// allAPIVersions records all AVIVersions that are supported at /apis, in the
+	// form of "group/version"
+	allAPIVersions := []string{}
```

Comment:
@smarterclayton @lavalamp, I move allAPIVersions into init(). When Brendan's PR is in, we can pass allAPIVersions to 3rd party controller so it can add/delete versions from it.